.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why is this VS config file here?

Reply:
To be honest I don't know. He is on my .gitignore. Gotta remove for the next commit